src/content/docs/logs/forward-logs/kubernetes-plugin-log-forwarding.mdx

```diff
@@ -43,76 +57,104 @@ To forward your Kubernetes logs to New Relic with our plugin:
+3. Import the resulting dashboard in JSON format by following [these instructions](/docs/query-your-data/explore-query-data/dashboards/dashboards-charts-import-export-data/#import-json).
 
-We capture Fluent Bit's internal metrics by using its [prometheus_scrape INPUT plugin](https://docs.fluentbit.io/manual/pipeline/inputs/prometheus-scrape-metrics) in conjunction with its [prometheus_remote_write OUTPUT plugin](https://docs.fluentbit.io/manual/pipeline/outputs/prometheus-remote-write). All the Prometheus `counter` metrics are actually _cumulative counters_, but we automatically perform a _delta conversion_ when they are ingested at New Relic to ease querying them using NRQL later. You can find more details [here](/docs/data-apis/understand-data/metric-data/cumulative-metrics/).
+4. The imported dashboard will be available in your [Dashboards page](https://one.newrelic.com/dashboards) as `Kubernetes Fluent Bit monitoring`.
+
+### Additional metric details [#addition-metric-details]
+
+The `newrelic-logging` Helm chart uses [Fluent Bit](https://fluentbit.io/) together with New Relic's [`newrelic-fluent-bit-output` plugin](https://github.com/newrelic/newrelic-fluent-bit-output) to send logs to New Relic. The `fluentBit.sendMetrics` configuration option enables the collection of their individual metrics:
+
+  * **[Fluent Bit internal metrics](https://docs.fluentbit.io/manual/administration/monitoring#for-v2-metrics)**: emitted by Fluent Bit in Prometheus format and delivered to New Relic's Prometheus export endpoint. They can be faceted by `cluster_name`, `node_name` and `pod_name`.
+
+  * **[`newrelic-fluent-bit-output`'s internal plugin metrics](https://github.com/newrelic/newrelic-fluent-bit-output?tab=readme-ov-file#troubleshooting-metrics)**: collected by the output plugin and sent to New Relic's metric API. These metrics only contain the `cluster_name` dimension, so they can be narrowed down to a particular cluster but not to a particular host or pod. They're useful to assess the overall latency when delivering the logs to the New Relic Logs API or to observe potential packaging problems.
+
+We capture Fluent Bit's internal metrics by using its [`prometheus_scrape` input plugin](https://docs.fluentbit.io/manual/pipeline/inputs/prometheus-scrape-metrics) in conjunction with its [`prometheus_remote_write` output plugin](https://docs.fluentbit.io/manual/pipeline/outputs/prometheus-remote-write). All the Prometheus `counter` metrics are actually cumulative counters, but we automatically perform a delta conversion when they are ingested at New Relic to ease querying them using NRQL later. You can find more details [here](/docs/data-apis/understand-data/metric-data/cumulative-metrics/).
 
 ## View log data [#find-data]
```

Comment:
@nbaenam maybe this should precede the troubleshooting section?

Reply:
I think this paragraph must be here because it refers to Fluent Bit's internal metrics. I've moved inside **[`newrelic-fluent-bit-output`'s internal plugin metrics](https://github.com/newrelic/newrelic-fluent-bit-output?tab=readme-ov-file#troubleshooting-metrics)**